charts/apps/custom-app/chart/templates/deployment.yaml

```diff
@@ -48,7 +48,7 @@ spec:
         command:
           - /bin/sh
           - -c
-          - cp -r -n {{ $.Values.appconfig.path }}/* /tmp
+          - if [ -n "$(ls -A {{ $.Values.appconfig.path }})" ]; then echo "Copying data..." && cp -r -n {{ $.Values.appconfig.path }}/* /tmp && echo "Done Copying"; elif [ -d "{{ $.Values.appconfig.path }}" ]; then echo "Empty directory. Nothing to copy."; else echo "Directory does not exist" && cp -r -n {{ $.Values.appconfig.path }}/* /tmp;  fi
```

Comment:
Lets rewrite this using the `|` operator. This is hard to read. 

```
  - |
    if [ -n "$(ls -A {{ $.Values.appconfig.path }})" ]; then
      echo "Copying data..." && cp -r -n {{ $.Values.appconfig.path }}/* /tmp && echo "Done Copying";
    elif [ -d "{{ $.Values.appconfig.path }}" ]; then
      echo "Empty directory. Nothing to copy.";
    else
      echo "Directory does not exist" && cp -r -n {{ $.Values.appconfig.path }}/* /tmp;
    fi
```

Reply:
Done!